src/styles/normalize.scss

```
@@ -0,0 +1,9 @@
+body {
+  margin: 0;
+  font-family: Arial, sans-serif;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vw;
+  font-size: 30px;
```

would be better move it to the `html` tag
```suggestion
  display: flex;
  align-items: center;
  justify-content: center;
  height: 100vw;
```